script/client

```diff
@@ -138,11 +147,32 @@ else {
 }
 
 $url->path($path);
-$url->query([%params]) if %params;
+
+if (!$options{'form'}) {
+    $url->query([%params]) if %params;
+}
+else {
+    my %form;
+    for (keys %params) {
+        if (/(\S+)\.(\S+)/) {
+            $form{$1}{$2} = $params{$_};
+        }
+        else {
+            $form{$_} = $params{$_};
+        }
+    }
+    %params = %form;
+}
```

Comment:
huh? what do you want to simplify? And what is 'gota'? https://en.wikipedia.org/wiki/Gota gives me too many options